src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.tag;
+
+import java.util.Objects;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Logs the addition/removal of tags during a session 
+ * Association class of Person and Tag
+ */
+public class Tagging {
+    public final Person person;
+    public final Tag tag;
+    public final boolean isAdd;
+    
+    /**
+     * Constructor
+     * Create a tagging
+     * @param person The tagged person
+     * @param tag The targeted tag
+     * @param isAdd true is the tag was added, false if the tag was removed from the person
+     */
+    public Tagging(Person person, Tag tag, boolean isAdd) {
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.